ui/components/app/confirm/footer/footer.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import configureStore from '../../../../store/store';
+import mockState from '../../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../../test/jest';
+import { Footer } from '.';
+
+const render = () => {
+  const store = configureStore({
+    metamask: {
+      ...mockState.metamask,
+    },
+    confirm: {
+      currentConfirmation: {
+        msgParams: {
+          from: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+        },
+      },
+    },
+  });
+
+  return renderWithProvider(<Footer />, store);
```

Comment:
how is the provider used in the test?

Reply:
If you mean we could just use rtl render here, yes we can...but this component uses I18N and we will have to pass that provider manually here if using RTL, I think it's easier to use our util renderWithProvider that already includes this.

Reply:
oh I see now. I think we can remove the mock store logic and call 
`renderWithProvider(<Footer />);` directly rather than `renderWithProvider(<Footer />, store);`